### ISSUE TITLE
Large .dat files written to Temp directories (Windows)

### ISSUE THREAD

**Issue by Haroon-Khel** [adoptium/infrastructure#2674]
On [test-azure-win2012r2-x64-1](https://ci.adoptopenjdk.net/computer/test-azure-win2012r2-x64-1/), [test-azure-win2012r2-x64-3](https://ci.adoptopenjdk.net/computer/test-azure-win2012r2-x64-3/) and [test-azure-win2019-x64-1](https://ci.adoptopenjdk.net/computer/test-azure-win2019-x64-1/) their Temp directories are filled up with large .dat files, whose origin is unknown

[test-azure-win2012r2-x64-1](https://ci.adoptopenjdk.net/computer/test-azure-win2012r2-x64-1/)
```
adoptopenjdk@test-2012r2-1 /cygdrive/c/Users/jenkins.test-2012r2-1/AppData/Local/Temp
$ ls -la *.dat
-rwx------+ 1 jenkins None 2147484671 Oct 10  2021 dst12191381147079577745.dat
-rwx------+ 1 jenkins None 2147484671 May  8 03:13 dst8808757851773313999.dat
-rwx------+ 1 jenkins None 2147484671 Oct 10  2021 src10083073893585810729.dat
-rwx------+ 1 jenkins None          0 Jun 19 01:10 src12523888732483928706.dat
-rwx------+ 1 jenkins None          0 Jun 26 01:35 src14450829699699143094.dat
-rwx------+ 1 jenkins None          0 Jul  3 02:00 src16185337324852676813.dat
-rwx------+ 1 jenkins None          0 Jul 10 22:30 src17095945012730825308.dat
-rwx------+ 1 jenkins None 2147484671 May  8 03:13 src17662931351367789633.dat
-rwx------+ 1 jenkins None 2147484671 May  8 04:37 src17862466297894479515.dat
-rwx------+ 1 jenkins None          0 Jul 10 02:01 src17894327494747906074.dat
-rwx------+ 1 jenkins None          0 Jun 26 05:03 src5264114818175128872.dat
-rwx------+ 1 jenkins None          0 Jul 10 04:11 src7319034148243782179.dat
```

[test-azure-win2012r2-x64-3](https://ci.adoptopenjdk.net/computer/test-azure-win2012r2-x64-3/)

```
AdoptopenJDK@test-2012r2-3 /cygdrive/c/Users/jenkins/AppData/Local/Temp
$ ls -la *.dat
-rwx------+ 1 jenkins None 2147484671 Jul  3 18:26 dst10525283875402494967.dat
-rwx------+ 1 jenkins None 2147484671 Jan  9  2022 dst1090982662380210025.dat
-rwx------+ 1 jenkins None 2147484671 Jan  9  2022 dst14090557438999081470.dat
-rwx------+ 1 jenkins None 2147484671 Oct 17  2021 dst15451742831967023501.dat
-rwx------+ 1 jenkins None 2147484671 May  8 02:44 dst16767680786618213670.dat
-rwx------+ 1 jenkins None 2147484671 Jul  3 00:59 dst5560256765629115735.dat
-rwx------+ 1 jenkins None 2147484671 Jul  4 00:14 src11220301557628620189.dat
-rwx------+ 1 jenkins None 2147484671 Jul  3 18:26 src12371769305297563754.dat
-rwx------+ 1 jenkins None 2147484671 Jul  3 00:59 src1798870126431615614.dat
-rwx------+ 1 jenkins None 2147484671 Jul 10 21:06 src5646674978054390026.dat
```

[test-azure-win2019-x64-1](https://ci.adoptopenjdk.net/computer/test-azure-win2019-x64-1/)
```
adoptopenjdk@test-win2019-1 /cygdrive/c/Users/jenkins/AppData/Local/Temp
$ ls -la *.dat
-rwx------+ 1 jenkins None 5368709122 Feb 7 01:57 LargeGatheringWrite3322105129955012357.dat
-rwx------+ 1 jenkins None 5368709122 Feb 20 08:19 LargeGatheringWrite9933378702831300867.dat
-rwx------+ 1 jenkins None 2147484671 Mar 14 09:43 dst12651648640093188310.dat
-rwx------+ 1 jenkins None 2147484671 Feb 6 10:31 dst183570549372488362.dat
-rwx------+ 1 jenkins None 2147484671 Jun 19 21:07 dst4338419869657228497.dat
-rwx------+ 1 jenkins None 2147484671 Feb 20 08:22 dst5702335999286369047.dat
-rwx------+ 1 jenkins None 2147484671 Jun 19 05:08 dst7728209470019621345.dat
-rwx------+ 1 jenkins None 2147484671 Feb 7 02:00 dst9261951980099793202.dat
-rwx------+ 1 jenkins None 2147484671 Jun 19 21:06 src14810603928291401966.dat
-rwx------+ 1 jenkins None 2147484671 Jun 19 05:06 src2088681814118365386.dat
```

Thie files are 2G each, often causing disk space issues on our machines. I dont think these files are important since theyre created in the Temp directory. Would be good to find their cause

ref https://github.com/adoptium/infrastructure/issues/2493#issuecomment-1185420927

**Comment by zdtsw** [comment]
can open these .dat files and see what's inside ? maybe can find out which program generated these files on windows?

**Comment by Haroon-Khel** [comment]
Nothing seems to open it. `cat`ing it on the command line reveals what looks like a binary file?

**Comment by sxa** [comment]
TL;DR - They will be safe to delete.

They'll be from one of the tests in the extended.openjdk suite. See https://github.com/adoptium/infrastructure/issues/2448 for comparison where an AIX machine's `/tmp` needed to be expanded. The file name format looks similar so it's likely the same test that's causing it here, and for whatever reason (aborts maybe?) it occasionally hasn't cleared up after itself on those windows machines.

**Comment by sxa** [comment]
We should also confirm if this is still the case on our Win2022 machines.
